respec/Inleiding.md

```diff
@@ -25,6 +40,7 @@ Het profiel dient tenminste aan de volgende richtlijnen te voldoen:
 - Laagdrempelig door modulair opbouw en duidelijk in vertalingen en uitleg.
 - We beginnen met begrippen, de overeenkomstige behoefte binnen de diverse deelnemende organisaties ligt voor een groot deel bij begrippen, dit is het laaghangende fruit voor een overeengekomen profiel.
 - Doel van het profiel is in eerste instantie om begrippen binnen organisaties kenbaar te maken.
```

Comment:
Wat willen we hiermee zeggen?
- Waarom "in eerste instantie"? Wanneer is de tweede instantie en wat gaan we dan doen?
- Het profiel is een specificatie met als doel interoperabiliteit van begrippencatalogi (BC). Een BC zou als doel kunnen hebben om begrippen kenbaar te maken, maar ons profiel toch niet?
- Waarom "binnen organisaties"? Niet erbuiten?